engine/api/workflowtemplate/instance.go

```diff
@@ -72,7 +72,9 @@ func requestModifyDefaultNameAndRepositories(ctx context.Context, db gorp.SqlExe
 		var repoPath string
 	loopVCSServer:
 		for _, vcs := range proj.VCSServers {
-			repos, err := repositoriesmanager.GetReposForProjectVCSServer(ctx, db, store, proj, vcs.Name, repositoriesmanager.Options{})
+			repos, err := repositoriesmanager.GetReposForProjectVCSServer(ctx, db, store, proj, vcs.Name, repositoriesmanager.Options{
```

Comment:
you cannot do it every time, in a sql transaction. It will timeout

Reply:
Resync was moved to cdsctl workflow init.